draft-ietf-gnap-core-protocol.md

```diff
@@ -4709,13 +4229,13 @@ for feedback and development of early versions of the XYZ protocol that fed into
 
 # IANA Considerations {#IANA}
 
-[[ TBD: There are a lot of items in the document that are expandable
-through the use of value registries. ]]
+\[\[ TBD: There are a lot of items in the document that are expandable
```

Comment:
Should we generate an issue for the TBD items?

Reply:
It doesn't hurt, but these are all very general TBD's that every draft spec needs to finish. We don't need to have linked issues for them, at least.